### PR TITLE
Remove jshint/jscsrc leftovers

### PR DESCRIPTION
**Pull request by gilligan**
Remove config files and jshint switches in
the sources.

### CODE REVIEW

File: test/bind.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-/* jshint -W053 */
```
Comment (davidchambers) [inline test/bind.js]: :space_invader:

**Comment by Siilwyn** [comment]
What's up with this?

**Comment by davidchambers** [comment]
We should merge this, I would say. :)

**Comment by CrossEye** [comment]
yup

**Pull request merged**